actionview/lib/action_view/template/handlers/erb.rb

```diff
@@ -58,6 +58,7 @@ def add_expr_escaped(src, code)
         def add_stmt(src, code)
           flush_newline_if_pending(src)
           super
+          flush_newline_if_comment(src, code)
```

Comment:
What about a test that just focuses on the `add_stmt` method, e.g.
```erb
<% # comment %>
<% nil %>
<% nil %>
<% nil %>
<% nil %>
```
Repeated 1000s of times?